src/style.css

```
@@ -1 +1,52 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.section {
+  width: 96px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 4px;
+}
+
+.stars__star {
+  display: inline-block;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  padding-right: 4px;
+}
+
+.stars__star:last-child {
+  padding-right: 0;
+}
```

to make outer indent you should use margin
and you can use `:not(:last-child)`
```suggestion
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```